internal/locale/locales/en-us.yaml

```diff
@@ -1835,6 +1835,13 @@ unstable_command_warning:
     This feature [ACTIONABLE]state {{.V0}}[/RESET] is still in beta. If you want to opt-in to unstable features, run the following command:
 
     [ACTIONABLE]state config set optin.unstable true[/RESET]
+err_update_corrupt_install:
```

Comment:
I don't think we actually use this, since the PR moved from erroring out to actually addressing the error. Though maybe we should print this as a fallback when the automatic cleanup fails.